applications/mne_analyze/libs/anShared/Management/event.h

```diff
@@ -115,9 +121,11 @@ class ANSHAREDSHARED_EXPORT Event : public QObject
 
     //=========================================================================================================
     /**
-    * @brief Destructor
+    * Getter for Event data.
+    *
+    * @return Data of the Event.
     */
-    ~Event();
+    inline QVariant getData() const;
```

Comment:
Should this be a reference to a QVariant ?

Reply:
No, returning a reference is dangerous because it could dangle.